core/src/main/kotlin/com/justai/jaicf/helpers/kotlin/Extensions.kt

```diff
@@ -0,0 +1,3 @@
+package com.justai.jaicf.helpers.kotlin
+
+internal fun <T> Boolean.runIfTrueElseNull(function: () -> T?): T? = if (this) function.invoke() else null
```

Comment:
Let's think about a shorter name as there are many cases on using it, but the name is neraly as much long as function's body. Moreover this function is internal.
What about just `ifTrue`?